src/ziggurat/timestamp_transformer.clj

```diff
@@ -17,17 +17,20 @@
                (get-current-time-in-millis)
                ingestion-time))))
 
-(deftype TimestampTransformer [^{:volatile-mutable true} processor-context metric-namespace oldest-processed-message-in-s] Transformer
+(deftype TimestampTransformer [^{:volatile-mutable true} processor-context metric-namespace oldest-processed-message-in-s topic-entity-name] Transformer
          (^void init [_ ^ProcessorContext context]
            (do (set! processor-context context)
                nil))
          (transform [_ record-key record-value]
            (let [message-time (.timestamp processor-context)]
              (when (message-to-process? message-time oldest-processed-message-in-s)
-               (calculate-and-report-kafka-delay metric-namespace message-time)
+               (calculate-and-report-kafka-delay metric-namespace message-time topic-entity-name)
```

Comment:
What's the metric namespace value here?

Reply:
the metric namespace value is whatever that is passed to the TimestampTransformer constructor

Reply:
whatever value you passed to TimestampTransformer